### DISCUSSION TITLE
adding damage to openfast

### DISCUSSION THREAD

**Discussion by lt72884** [OpenFAST/openfast#1880] [Q&A] [answered]
Is it possible to add a crack or some sort of damage to openfast to see how it effects the life of the blade via MLife?

thanks

**Comment by jjonkman** [comment, marked as answer]
Dear @lt72884,

See a similar question asked and answered on our forum: https://forums.nrel.gov/t/making-fault/1260.

Best regards,

**Comment by lt72884** [comment]
thank you for that :)
